app/src/main/res/values-be/strings.xml

```diff
@@ -20,6 +20,20 @@
     <string name="unpin_conversation">Адмацаваць</string>
     <string name="forward_message">Пераслаць</string>
     <string name="compress_error">Немагчыма сціснуць выяву да выбранага памеру</string>
+    <!-- vCard-->
+    <plurals name="view_contact">
```

Comment:
where and how is and_other_contacts used? Maybe we could replace view_contact with view_contact_details from Simple Commons, no need to check plurals then

Reply:
**"View contact" vs "View contact details":**
<img src="https://user-images.githubusercontent.com/36371707/187162875-78655149-4fb8-4df5-9bbc-aa2599d0a62e.png" width=264 /> <img src="https://user-images.githubusercontent.com/36371707/187163415-f4e3440d-c27d-44b2-aea7-d79bdc15dbd7.png" width=264 />

looks fine to me.

> where and how is and_other_contacts used?

as shown in the screenshot, it's used to display the number of contacts in the vcard. That's how they did it in Samsung messages.

**Multiple contacts in Google messages:**
<img src="https://user-images.githubusercontent.com/36371707/187164483-4bb20449-0656-4b6f-9c55-a138786b7c37.png" width=264 />

I chose samsung way as it provides more info to the user without clicking

Reply:
alright, this looks good, we can keep it